sns/src/main/scala/akka/stream/alpakka/sns/javadsl/SnsPublishSink.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.sns.javadsl
+
+import java.util.concurrent.CompletionStage
+
+import akka.Done
+import akka.stream.alpakka.sns.SnsPublishSinkStage
+import akka.stream.javadsl.{Sink => JSink}
+import akka.stream.scaladsl.Sink
+import com.amazonaws.services.sns.AmazonSNSAsync
+
+import scala.compat.java8.FutureConverters._
+
+object SnsPublishSink {
```

Comment:
Maybe this should then instead be `SnsPublisher` and have both `.sink` and `.flow` in the same place?